IowaPD_dispatch.yaml

```
@@ -1,7 +1,9 @@
+# select * from dispatch_all;
+
 entityDefinitions:
-  JohnsonCounty_IowaCity_Dispatch:
-    fqn: ol.dispatch
-    entitySetName: JohnsonCounty_IowaCity_Dispatch
+  JohnsonCounty_IowaCity_CallsForService:
```

Can we shorten the names a bit?
```suggestion
  JohnsonCounty_CallsForService:
```